yt/yt/library/auth_server/oauth_service.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "public.h"
+
+#include <yt/yt/core/actions/future.h>
+#include <yt/yt/core/ytree/public.h>
+
+namespace NYT::NAuth {
+
+////////////////////////////////////////////////////////////////////////////////
+
+struct TOAuthUserInfoResult {
+    TString Sub;
```

Comment:
What is Sub?

Reply:
The sub (subject) property, typically found in response data, carries the unique identifier of the user who has logged in. It can be a valuable piece of information. However, it's not always present in the sub field.

To handle this, we might consider allowing the field name to be set in the configuration and saved. We could add an optional field in the configuration; if set, the value will be stored.

Alternatively, we could choose to remove the `sub` from the `TOAuthUserInfoResult` since it's not utilized in the cookie authenticator.

Reply:
At least I would prefer to use full name for this variable `Subject`